text/2024-06-21-combatting-fuzzer-timeouts.md

```diff
@@ -72,3 +76,16 @@ There are two broad classes of mechanisms that can cause non-linear scaling in p
    lookups visited and end processing when a configured limit is reached. In CFF/CFF2 processing, we limit the number of instructions that
    can be executed. In Harfbuzz, [hb-limits.h](https://github.com/harfbuzz/harfbuzz/blob/main/src/hb-limits.hh) contains several
    configurable operation limits. This can be used as a starting point to find places where we've implemented operation counts.
+   Additionally, we also define some more limits in
+   [hb-subset-plan.cc](https://github.com/harfbuzz/harfbuzz/blob/main/src/hb-subset-plan.cc#L799).
```

Comment:
We should move that to `hb-limits` I think. Seems unintentional

Reply:
sounds good.

Reply:
https://github.com/harfbuzz/harfbuzz/pull/4759